### PR TITLE
Fix when trying to publish an HD simulcast with only 2 layers set.

### PR DESCRIPTION
**Pull request by kekkokk**
When trying to publish an HD stream with only 2 layers set, due to a chrome implementation [Chromium Source](https://chromium.googlesource.com/external/webrtc/+/branch-heads/51/webrtc/media/engine/simulcast.cc#42), nothing is sent and results in a empty video on the subscriber side.

It alerts the publisher and automatically set layers to 3 (which is the minimum for an HD stream).

[] It needs and includes Unit Tests
[] It includes documentation for these changes in `/doc`.

### CODE REVIEW

**Comment by kekkokk** [comment]
updated with new Room.js

**Comment by kekkokk** [comment]
!up please merge or close

File: erizo_controller/erizoClient/src/Room.js [hunk 1]
```diff
@@ -587,6 +587,14 @@ Erizo.Room = (specInput) => {
 
     options.simulcast = options.simulcast || false;
 
+    if (stream.videoSize && stream.videoSize[0] > 640 && options.simulcast &&
+      options.simulcast.numSpatialLayers <= 2) {
+      L.Logger.warning('Trying to publish an HD stream with only 2 quality layers. Due to a Chrome ' +
```
Comment (jcague) [inline erizo_controller/erizoClient/src/Room.js]: a warning might be too high cause it's a bug in Chrome, that might be fixed at some point. I agree with the log, but I'd rather use L.logger.debug

Comment (jcague) [inline erizo_controller/erizoClient/src/Room.js]: Please, move the code to ChromeStableStack since it only affects Chrome.

**Comment by jcague** [comment]
@kekkokk did you have some time to see the comments? what do you think?

**Comment by kekkokk** [comment]
yeah sorry, I'll fix it asap

**Comment by kekkokk** [comment]
!

File: erizo_controller/erizoClient/src/webrtc-stacks/ChromeStableStack.js [hunk 2]
```diff
@@ -3,9 +3,18 @@ import SdpHelpers from './../utils/SdpHelpers';
 import Logger from '../utils/Logger';
 
 const ChromeStableStack = (specInput) => {
+  const newSpec = specInput;
   Logger.info('Starting Chrome stable stack', specInput);
-  const spec = specInput;
-  const that = BaseStack(specInput);
+  if (newSpec.videoSize[0] > 640 && newSpec.simulcast &&
```
Comment (jcague) [inline erizo_controller/erizoClient/src/webrtc-stacks/ChromeStableStack.js]: what happens if newSpec.videoSize is undefined?

File: erizo_controller/erizoClient/src/webrtc-stacks/ChromeStableStack.js [hunk 3]
```diff
@@ -3,9 +3,18 @@ import SdpHelpers from './../utils/SdpHelpers';
 import Logger from '../utils/Logger';
 
 const ChromeStableStack = (specInput) => {
+  const newSpec = specInput;
```
Comment (jcague) [inline erizo_controller/erizoClient/src/webrtc-stacks/ChromeStableStack.js]: why not changing spec instead of creating newSpec and reassigning it?

**Pull request closed without merging**